pkg/popeye.go

```diff
@@ -34,6 +36,8 @@ var (
 	LogFile = filepath.Join(os.TempDir(), "popeye.log")
 	// DumpDir indicates a directory location for sanitizer reports.
 	DumpDir = dumpDir()
+	// ErrUnknownS3BucketProtocol defines the error if we can't parse the S3 URI
+	ErrUnknownS3BucketProtocol = errors.New("invalid S3 URI: hostname not valid")
```

Comment:
Is this sentinel error necessary? Don't think there is much reuse for it as it is defined in a very specific context.

Reply:
I kept the sentinel error to help with testing, would that be ok or would you rather me still remove it?